.changelog/3108.breaking.1.md

```diff
@@ -0,0 +1,3 @@
+go/oasis-test-runner/cmd: Rename `--test` flag to `--scenario` flag
```

Comment:
Make sure to update all the CI pipelines, I don't see any here and I know there are places that use `-t`.

Reply:
Thanks for the pointer, I think I fixed all these.